Documentation/Upgrade/README.rst

```diff
@@ -0,0 +1,115 @@
+.. _README:
+
+.. include:: ../lib/doc-includes/pike-icons.inc
+
+.. Don't use default python highlighting for code blocks http://www.sphinx-doc.org/en/stable/markup/code.html
+
+.. highlight:: none
+
+
+===================================
+Minor Upgrade to Release 5.4.1 U8
+===================================
+
+.. contents::
+   :local:
+   :depth: 3
+
+
+Use this documentation when upgrading between minor releases. The process applies to upgrades from Release 5.4.1 U5 to Release 5.4.1 U8+. During this process, the Nuage components can be updated at the same time.
+
+It is assumed the operator is familiar with OpenStack upgrades, VSP installation, the distribution-specific installation and upgrade practices, and the specific requirements for operations in a production environment.
+
+
+Upgrade Paths
+-------------
+
+In this release, you can upgrade only from Queens 5.4.1 U5 to Queens 5.4.1 U8.

```

Comment:
That is what we agreed to QA, but i believe we claim U6 to U8 also.  So this statement is too limited to my view.

Reply:
See line 46 also where we say "or later"

Reply:
updated line 27